recipes/restraintlib/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set version = "2021.7.7" %}
+{% set sha256 = "e3d07556034aa0bfffcff55c6051aa55099c8196a2c4ab77450f9a872863d376" %}
+
+package:
+  name: restraintlib
+  version: {{ version }}
+
+source:
+  url: https://github.com/mkowiel/restraintlib/archive/refs/tags/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Since this looks to be pure Python you should be able to make this `noarch: python`, which will really cut down on the number of builds we need to do:
```suggestion
  number: 0
  noarch: python
```
You'll also need to add Python version constraints (e.g. `>=3.6`) below under `host` and `run`